services/movie_session.py

```diff
@@ -42,3 +42,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    list_result = []
+    for values in (MovieSession.objects
+                               .filter(id=movie_session_id)
+                               .values_list("ticket__row", "ticket__seat")):
```

Comment:
```suggestion
    for values in (
        MovieSession.objects
         .filter(id=movie_session_id)
         .values_list("ticket__row", "ticket__seat")
    ):
```
Так трошки зрозуміліше яка душка куди

Reply:
(рекомендація)